easybuild/easyconfigs/m/Mono/Mono-4.6.2.7.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mono'
+version = '4.6.2.7'
+
+homepage = 'http://mono-framework.com' 
+description = """An open source, cross-platform, implementation of C# and the CLR that is
+ binary compatible with Microsoft.NET."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://download.mono-project.com/sources/mono/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+osdependencies = ['gettext']
```

Comment:
Well, I added it as an osdependency because it worked with the version of gettext that was installed as a package from the os, and I didn't want to add yet another module to our already overloaded module list. But that said, feel free to add a proper EasyBuild dependency for the sake of reproducability...